src/RemoteAPI/class-remote-api-cache.php

```diff
@@ -54,7 +54,11 @@ public function __construct( Remote_API_Base $remote_api, Cache $cache ) {
 	 *                                             response is empty.
 	 */
 	public function get_items( $query, $associative = false ) {
-		$cache_key = 'parsely_api_' . wp_hash( (string) wp_json_encode( $this->remote_api ) ) . '_' . wp_hash( (string) wp_json_encode( $query ) );
+		$cache_key = (
```

Comment:
Out of curiosity, what's the reason for using parentheses for this assignment?

Reply:
No reason. It could have been avoided.

Reply:
OK, we can leave as is for now.